package.json

```diff
@@ -18,19 +18,37 @@
   },
   "homepage": "https://github.com/aligent/aws-cdk-constructs#readme",
   "scripts": {
-    "build": "tsc",
-    "lint": "eslint --ignore-path .eslintignore --ext .ts .",
-    "lint:check": "npm run lint",
-    "lint:fix": "npm run lint --fix",
+    "build": "nx exec -- tsc",
+    "lint": "eslint . --ignore-pattern '.pnp.*'",
```

Comment:
As above, we shouldn't call eslint, prettier etc ourselves but instead through the NX targets / NX format commands.